cmd/skipper/main.go

```diff
@@ -1,11 +1,37 @@
+// Copyright 2015 Zalando SE
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 // skipper program main
 //
 // for a summary about skipper, please see the readme file.
+
+/*
+Command skipper provides a default executable skipper with the default
+set of filters.
+
```

Comment:
Maybe have some reference, somewhere, which ones? Can we link to specific code parts/functions?